### PR TITLE
feat: Spanish translations

### PR DESCRIPTION
**Pull request by elypke**
**Summary**
Spanish translations

**Testing**
Validate syntax and grammatic

### CODE REVIEW

**Comment by scnwwu** [comment]
Hi, thank you for contribution! But the change looks not correct. There should not be `bundle.l10n.es.json` file under root directory. The `l10n/bundle.l10n.es.json` file contains garbage characters. The key of the json should remain in English and the translation should be in the value part of the json. Please refer to other locales. Thanks.

**Comment by scottdover** [comment]
[es-update-feb2024.csv](https://github.com/sassoftware/vscode-sas-extension/files/14103339/es-update-feb2024.csv)

@elypke If it's helpful, you can update this spreadsheet and re-attach it to the PR instead of worrying about json files. You'll just need to update everything in column C

**Comment by elypke** [comment]
@scnwwu @scottdover Hi guys, thank you for your comments, 

I have added the file bundle.l10n.es

@scottdover I have just read your comment recently,  just in case I will update your file and i will send you

**Comment by scnwwu** [comment]
Hi @elypke, there're some updates about the messages in main branch. Would you please update?
And please make sure to have LF line ending to pass the build.
If any problems, @scottdover might be able to provide you an updated csv file to update. Thanks.

**Comment by elypke** [comment]
Hi @scnwwu @scottdover 
I'm sending you the updated file
Regards 
[es-update-feb2024.csv](https://github.com/sassoftware/vscode-sas-extension/files/14211967/es-update-feb2024.csv)

File: l10n/bundle.l10n.es.json [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  "Python Program": "Python Program",
+  "Rename File": "Renombrar archivo",
+  "Rename Folder": "Renombrar carpeta",
+  "Result": "Resultado",
+  "Result: {result}": "Resultado: {result}",
+  "SAS 9 Server": "Servidor SSH SAS 9",
+  "SAS Compute Context": "Contexto de cálculo SAS",
+  "SAS Log": "Log SAS",
+  "SAS Profile": "Perfil SAS",
+  "SAS Program": "SAS Program",
```
Comment (scottdover) [inline l10n/bundle.l10n.es.json:81]:
```suggestion
  "SAS Program": "Programa SAS",
```

File: l10n/bundle.l10n.es.json [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  "SAS Server Username": "Nombre de usuario del servidor SAS",
+  "SAS Viya Server": "Servidor SAS Viya",
+  "SAS code running...": "Código SAS en ejecución...",
+  "SQL Program": "SQL Program",
```
Comment (scottdover) [inline l10n/bundle.l10n.es.json:85]:
```suggestion
  "SQL Program": "Programa SQL",
```

File: l10n/bundle.l10n.es.json [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+  "No Profiles available to delete": "No hay ningún perfil disponible para eliminar",
+  "No SAS Connection Profile": "No hay un perfil de conexión SAS",
+  "No authorization code": "No hay ningún código de autorización",
+  "No opened file": "No hay un archivo abierto",
+  "No valid sas code": "No hay un código sas válido",
+  "Not a valid sas file: {file}": "No es un archivo sas válido: {file}",
+  "Not implemented": "No implementado",
+  "Paste authorization code here": "Pegar aquí el código de autorización",
+  "Port Number": "Número de puerto",
+  "Python Program": "Python Program",
```
Comment (scottdover) [inline l10n/bundle.l10n.es.json:72]:
```suggestion
  "Python Program": "Programa Python",
```

**Review by scottdover — COMMENTED** [review]
Hey @elypke. Thanks for the csv updates! I've imported those and have updated the pull request with the results.

There are still a few strings that need translation. I've added suggestions with results I've found from google translate. You can commit the suggestions, or manually edit them if the google translate results are wrong.

Also, do you mind updating `CONTRIBUTING.md` to include your name for Spanish translations? There should be a table under `### Locale contributors` that you can add to.

After those things are taken care of, I think this one is good to merge. Thanks so much for your contribution :)

**Review by scottdover — APPROVED** [review]

**Comment by scottdover** [comment]
Thank you for your contribution, @elypke 😊. It's appreciated. I realize I asked you to update the contributors table then merged this one in before that was done. Do you mind providing your first/last name and I'll get that updated?

**Comment by elypke** [comment]
Ok,
Elyana Mastache and Raquel Nunez

**Comment by scottdover** [comment]
Thank you!

**Pull request merged**